docs/languages/cpp.md

```diff
@@ -194,57 +143,77 @@ To go to a symbol's definition, place your cursor on the symbol anywhere its use
 
-### Function breakpoints
+### Function Breakpoints
 
-Function breakpoints enable you to break execution at the beginning of a function instead of on a particular line of code. To set a function breakpoint, on the __Debug Panel__, right click inside the __Breakpoints__ pane, then choose __Add Function Breakpoint__ and enter the name of the function on which you want to break execution.
+Function breakpoints enable you to break execution at the beginning of a function instead of on a particular line of code. To set a function breakpoint, on the __Debug__ pane, right click inside the __Breakpoints__ section, then choose __Add Function Breakpoint__ and enter the name of the function on which you want to break execution.
 
-### Expression evaluation
+### Expression Evaluation
 
 VS Code supports expression evaluation in several contexts:
 
-* You can type an expression into the __Watch__ pane and it will be evaluated each time a breakpoint is hit.
+* You can type an expression into the __Watch__ section and it will be evaluated each time a breakpoint is hit.
 * You can type an expression into the __Debug Console__ and it will be evaluated only once.
 * You can evaluate any expression that appears in your code while you're stopped at a breakpoint.
 
-Note that expressions in the Watch Pane take effect in the application being debugged; an expression that modifies the value of a variable will modify that variable for the duration of the program.
+Note that expressions in the __Watch__ section take effect in the application being debugged; an expression that modifies the value of a variable will modify that variable for the duration of the program.
 
-### Multi-threaded debugging
+### Multi-threaded Debugging
 
-The C/C++ extension for VS Code now has the ability to debug threaded code.
+The C/C++ extension for VS Code has the ability to debug multi-threaded programs. All threads and their call stacks appear in the __Call Stack__ section:
 
-### Core Dump debugging
+![Multi-threaded process](images/cpp/threads.png)
 
-The C/C++ extension for VS Code also has the ability to debug using a memory dump. To debug using a memory dump, open your launch.json file for editing and add the `coreDumpPath` property to the __C++ Launch__ configuration, setting its value to be a string containing the path to the core dump. This will even work for multi-threaded programs and x86 programs being debugged on an x64 machine.
+### Memory Dump Debugging
 
-### GDB and MI commands
+The C/C++ extension for VS Code also has the ability to debug using a memory dump. To debug using a memory dump, open your `launch.json` file for editing and add the `coreDumpPath` (for GDB or LLDB) or `dumpPath` (for the Visual Studio Windows Debugger) property to the __C++ Launch__ configuration, setting its value to be a string containing the path to the memory dump. This will even work for multi-threaded programs and x86 programs being debugged on an x64 machine.

```

Comment:
I agree on both. That's how it was originally written and I just changed some words to generalize it to Windows and Unix dumps.